### PR TITLE
Add exporting

### PR DESCRIPTION
**Pull request by ConnorJL**
Adds a new "--export" flag that exports the model into a SavedModel that outputs raw logits.

### CODE REVIEW

**Comment by sdtblck** [comment]
LGTM
